www/_template/guides/svelte.md

```diff
@@ -1,12 +1,294 @@
+
+> 💡 Tip: check out your network tab in the browser inspector- you'll see only the file that changed reloading, not all the files that make up the site.
+
+## Customize your project layout
+
+Since you'll be adding a bunch of new files, you probably don't want them crowding up your top-level root directly. Snowpack is flexible enough to support whatever project layout that you prefer. In this guide, you'll learn how to use a popular project pattern from the Svelte community.
+
+```
+📁 src : your Svelte components and their assets (CSS, images)
+    ↳ index.js
+    ↳ App.svelte
+📁 public : global assets like images, fonts, icons, and global CSS
+    ↳ index.css
+    ↳ index.html
+```
+
+Use your favorite visual editor to rearrange and rename, or run these commands in the terminal:
+
+```bash
+mkdir src
+mkdir public
+mv index.js src/index.js
+mv App.svelte src/App.svelte
+mv index.html public/index.html
+mv index.css public/index.css
+```
+
+This means if you are running Snowpack right now, the site is now broken as the files are all in different places. Lets add a "mount" configuration to update your site to your new project layout.
+
+The `mount` configuration changes where Snowpack scan for and builds files. Head back to the `snowpack.config.js` file you edited when you added `@snowpack/plugin-svelte`. Add this to the empty `mount` object:
+
+```diff
+  mount: {
+-   /* ... */
++   // directory name: 'build directory'
++   public: '/',
++   src: '/_dist_',
+  },
+```
+
+<img src="/img/guides/react/folderstructure.png" alt="The original file configuration had Snowpack building the directory structure the same as the directories in the project, including root. Now the config builds only src and public. Src to the dist folder and public to root." />
+
+`mount` is part of the [Snowpack Configuration API](https://www.snowpack.dev/#configuration). It allows you to customize the file structure of your project. The key is the name of the directory and the value is where you'd like them in the final build. With this new configuration, Snowpack builds files in `public` like `public/index.css` directory into `index.css`. It builds files in `src` like `src/index.js` into `/_dist_/index.js`, so you'll need to change that path in your `index.html`:
+
+```diff
+  <body>
+-   <h1>Welcome to Snowpack!</h1>
+-   <script type="module" src="/index.js"></script>
++   <script type="module" src="/_dist_/index.js"></script>
+  </body>
+```
+
+You'll need to restart Snowpack for configuration file changes. When you start up again, if it worked, it should look the same.
+
+## Styling your project
+
+Your site is a little plain, so in this step you'll add an image and animate it with CSS.
+
+> 💡 Tip: as you're doing this, you should not need to reload the page or restart Snowpack. Snowpack automatically updates the project in the browser as you edit code.
```

Comment:
You've already mentioned this a few paragraphs above and even had a gif. I think safe to not include this, and assume a reader will read that, see that gif, and/or notice it on their own.